cuegui/cuegui/Constants.py

```diff
@@ -32,7 +32,13 @@
 import opencue
 
 
-VERSION = "1.3.0"
+possible_version_paths = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), 'VERSION.in')
+if os.path.exists(possible_version_path):
+    with open(possible_version_path) as fp:
+        version = fp.read().strip()
+else:
+    version = 'UNKNOWN'
```

Comment:
For now let's fall back to the old value (`"1.3.0"`). It's not correct but it's ok for now -- it's not worse than what we have now.

Basically I am looking at what happens when someone does `setup.py install` to install CueGUI, and it looks like I may have to adjust our packaging pipeline so version information gets distributed correctly in that scenario. So in the meantime I want to avoid showing `UNKNOWN` to a bunch of users :)